docs/troubleshooting/semgrep.md

```diff
@@ -7,21 +7,21 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Troubleshooting Semgrep
 
-## Troubleshooting Semgrep CI/action/agent
+## Troubleshooting Semgrep CI
 
-If you're seeing results reported for files that weren't touched, Github actions timing out, or anything else related to running semgrep in CI, see instructions here based on your CI provider.
+If you're seeing results reported files that have not changed since the last scan, Github actions timing out, or anything else related to running semgrep in CI, see instructions in sections below on your CI provider.
```

Comment:
```suggestion
If you're seeing results reported files that have not changed since the last scan, GitHub Actions timing out, or anything else related to running semgrep in CI, see instructions in sections below on your CI provider.
```